test/config/package_json/test.js

```diff
@@ -10,7 +12,7 @@ describe('Config', () => {
 
       const rules = configuration.getRules();
 
-      assert.equal(rules.length, 1);
+      assert.equal(2, rules.length);
```

Comment:
I've changed this given that the configuration now has the customRules added to the same package.json. 

We could: 
1) mock the cosmic operation so we return an inline pacakge.json to isolate the tests from each other. 
2) leave it as is till becomes hard to manage

Reply:
I suppose another way could be to use `os.tmpdir()` to setup a `configDirectory` with a given `package.json`. We could have a test helper along the lines of:

```js
  const config = { ... };

  const configuration = new Configuration({
    configDirectory: temporaryConfigDirectory(config),
  });
```

I don't mind leaving this as-is in this PR though.

Reply:
lemme give it a go after we merge this PR. happy to make the necessary changes